### DISCUSSION TITLE
I can't start the game after I updated to v3.0.0.0

### DISCUSSION THREAD

**Discussion by PyaeSone98** [tsunamods-codes/7th-Heaven#136] [Q&A] [answered]
I also installed .net framework 7.0 and latest  visual C++ distributable (both x64 and x84 ). But when I start the game after mounting the virtual disc and this shows up "Unknown error when launching the game: The process cannot access the file 'c:\games\final fantasy vii\7thWrapperProxy.runtimeconfig.json' because it is being used by another process." I already tried what i can but it didn't fix the issue.  Please help me with this issue

**Comment by julianxhokaxhiu** [comment, marked as answer]
There has been an announcement on the Discord channels about this. But long story short see the instructions here: https://github.com/tsunamods-codes/7th-Heaven#install

I personally suggest to use the installer we provide as it is the easy path.
